### ISSUE TITLE
Prettier 1.4 deletes shebang

### ISSUE THREAD

**Issue by anmonteiro** [prettier/prettier#1890]
I have a file in which Prettier deleted the `#!/usr/bin/env node` shebang line.

I'm not sure what more info you need to diagnose this, please let me know what else I can provide!

**Comment by vjeux** [comment]
@josephfrazier, looks like your shebang fixed broke it :(

**Comment by anmonteiro** [comment]
Minimal repro:

```
$ cat x.js
#!/usr/bin/env node
'use strict';

var x = 42;
$ ./node_modules/.bin/prettier x.js
"use strict";

var x = 42;
```

**Comment by josephfrazier** [comment]
:( Oops, I'll take a look

**Comment by josephfrazier** [comment]
Hmm, looks like this has something to do with the CLI, as the API appears to be working correctly: https://runkit.com/josephfrazier/runkit-npm-prettier-issue1890

```js
var prettier = require("prettier")

prettier.format(`#!/usr/bin/env node
'use strict';

var x = 42;`)
```

```
#!/usr/bin/env node
"use strict";

var x = 42;
```

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
